src/Native/System.Security.Cryptography.Native/CMakeLists.txt

```diff
@@ -11,6 +11,12 @@ add_compile_options(-Wno-cast-align)
 
 add_definitions(-DPIC=1)
 
+if(CMAKE_STATIC_LIB_LINK)
+   message("Build OpenSSL Static Link")
+   set(CMAKE_FIND_LIBRARY_SUFFIXES .a)
+   set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -lcrypto -lz -ldl  -static-libgcc -static-libstdc++ -static")
```

Comment:
So, System.Net.Http.Native only static linked against libcurl; but System.Security.Cryptography.Native seems to be static linking against the world.  I'm also confused why -lcrypto is explicitly stated, since the libcrypto reference should have already been figured out (along with libssl).

I guess I'm also also confused why we're setting EXE linker flags, since we're not an EXE; but maybe that's just what the variable is called.

And the message is also a bit weird... we're not building OpenSSL...